client/app/components/form/widgets/ComboBox.tsx

```diff
@@ -10,6 +10,7 @@ const ComboBox: React.FC<WidgetProps> = (props) => {
   const handleChange = (e: React.ChangeEvent<{}>, option: any) => {
     onChange(option?.enum?.[0]);
   };
+  console.log("value", value, schema);
```

Comment:
Thanks, removed! 🙇‍♂️

Reply:
@Sepehr-Sobhani okay I have fixed the issues with the headers, extra province field and the provinces not being populated.

I haven't fixed the `NaN` as we discussed I fixed it in the multi step form PR. Also not worrying about the senior officer wireframe since I will have to reorganize the form for the multistep PR I will be working on soon so I will ensure it matches the wireframes.

Reply:
Sounds good! Thank you @marcellmueller ! 🙏